src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -127,6 +127,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@adityapatwardhan, your last commit had 36 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

ForEach-Object -Parallel -AsJob Basic Tests.Verifies terminating error in single iteration
```powershell
Expected exactly 0, but got $null.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1: line 277
277:         $results.Count | Should -BeExactly 0
```
Adapter Tests.Property Adapter Tests.Count and length property works for singletons
```powershell
Expected 0, but got $null.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/ETS/Adapter.Tests.ps1: line 126
126:             $null.Count | Should -Be 0
```
TabCompletion.Miscellaneous completion tests.Test [CommandCompletion]::GetNextResult
```powershell
Expected strings to be the same, but they were different.
Expected length: 13
Actual length:   6
Strings differ at index 1.
Expected: 'Configuration'
But was:  'Cmdlet'
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 854
854:             $res.GetNextResult($false).CompletionText | Should -BeExactly 'Configuration'
```
Break statements with classes.no classes involved.doesn't report parse error for non-existing label
```powershell
Expected 0, but got $null.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Language/Classes/Scripting.Classes.Break.Tests.ps1: line 143
143:             $errors.Count | Should -Be 0
```
Basic FileSystem Provider Tests.Validate basic FileSystem Cmdlets.Clear-Content of a file
```powershell
Expected 0, but got $null.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 178
178:             $contentAfter.Count | Should -Be 0
```